examples/flows_and_runs_tutorial.py

```diff
@@ -58,7 +58,8 @@
 ############################################################################
 # Share the run on the OpenML server
 #
-# So far the run is only available locally. By calling the publish function, the run is sent to the OpenML server:
+# So far the run is only available locally. By calling the publish function,
```

Comment:
Did you double-check that introducing line breaks here does not mess with the layout of the generated documentation? I would expect not but good to make sure.

Reply:
Yes, they do not change the layout. In fact, some examples were already formatted correctly.